src/components/dashboard/IQButton.tsx

```diff
@@ -1,11 +1,19 @@
-import { Box, Button, Tooltip, Spinner } from '@chakra-ui/react'
+import {
+  Box,
+  Button,
+  Tooltip,
+  Spinner,
+  ButtonProps,
+  useBreakpointValue,
+} from '@chakra-ui/react'
 import React, { useEffect, useState } from 'react'
 import { addToken } from '@/utils/add-new-token'
 import Image from 'next/image'
 import * as Humanize from 'humanize-plus'
 import axios from 'axios'
 
-export const IQButton = () => {
+export const IQButton = (props: ButtonProps) => {
+  const sigFigs = useBreakpointValue({ base: 4, md: 4 })
```

Comment:
The value does not change still, why not use 

`const sigFigs = useBreakpointValue({ base: 4 })`